src/etc/inc/captiveportal.inc

```diff
@@ -1231,9 +1241,18 @@ function captiveportal_disconnect_client($sessionid, $term_cause = 1, $logoutRea
 }
 
 /* remove all clients */
-function captiveportal_disconnect_all($term_cause = 6, $logoutReason = "DISCONNECT") {
+function captiveportal_disconnect_all($term_cause = 6, $logoutReason = "DISCONNECT", $carp_loop = false) {
 	global $g, $config, $cpzone, $cpzoneid;
 
+	if (xmlrpc_sync_get_details($syncip, $syncport, $syncuser, $syncpass, $carp_loop)) {
+		$rpc_client = new pfsense_xmlrpc_client();
+		$rpc_client->setConnectionData($syncip, $syncport, $syncuser, $syncpass);
+		$rpc_client->set_noticefile("CaptivePortalUserSync");
+		$arguments = array('term_cause' => $term_cause,
+		'logout_reason' => $logoutReason);
+
+		$rpc_client->xmlrpc_method('captive_portal_sync', array('op' => 'disconnect_all', 'zone' => $cpzone, 'arguments' => base64_encode(serialize($arguments))));
```

Comment:
This line is very long, either move the `array(...` part to a new line, or define the array before using it. I know we prefer to avoid single use variables but in this case it would make it much easier to read/follow.